class.jetpack-twitter-cards.php

```diff
@@ -36,6 +36,18 @@ static function twitter_cards_tags( $og_tags ) {
 		}
 
 		if ( ! is_singular() || ! empty( $og_tags['twitter:card'] ) ) {
+			/**
+			 * Filter the default Twitter card image, used when no image can be found in a post.
+			 *
+			 * @since 5.8
+			 *
+			 * @param string $str Default image URL.
+			 */
+			$image = apply_filters( 'jetpack_twitter_image_default', '' );
```

Comment:
What do you think about changing the filter name to `jetpack_twitter_cards_image_default` to match other filters in that file?